lib/src/widgets/custom_scroll_behavior.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class CustomScrollBehavior extends ScrollBehavior {
```

Comment:
I believe a more specific name would make more sense than just using custom as prefix.

Reply:
Do you have a recommendation?
`BottomPaddedScrollBehavior`?